cli-docker/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+# config file
+if [ -n "$DECODABLE_ACCOUNT_NAME" ]; then
```

Comment:
Instead of the env var business which is a bit touchy due to the history concerns you describe above, why not simply expect and accept a volume with the decodable config file?

Reply:
because the user has to then generate/create the file. it's additional friction. this is only an example and we flag the security risk.

Reply:
OK, gotcha. With the updated instructions above, it makes more sense to me now.